src/xlsx/xlsx_writer.ts

```diff
@@ -142,19 +143,26 @@ function createWorksheets(data: ExcelWorkbookData, construct: XLSXStructure): XL
 
     const images = sheet.images;
     for (const image of images) {
+      const contentType = image.data.contentType;
```

Comment:
> if we don't have a contentType, then we should just drop the addition of that image in excel altogether because there is a non negligeable chance that it will still corrupt the generated file.

For the images that are already uploaded before this PR, when calling `createWorksheets()`, `contentType` (now `mimetype` according to your first comment) in `Image` type doesn't exist. The mime type only exist in the database. 

If we don't export images without `mimetype`, old images will not be exported. Is it better to set the mimetype to be `image/jpeg` by default for old images? Only when the mimetype is not supported we won't export them.